main.go

```diff
@@ -341,6 +364,13 @@ func getSlabJSON(w http.ResponseWriter, r *http.Request) {
 }
 
 func main() {
+
+	host := flag.String("host", "localhost", "identifier for ipfs and ipfs cluster")
+	port := flag.String("port", "9094", "Source port number")
```

Comment:
Shouldn't the description be "Port number of ipfs-cluster" instead?

Reply:
corrected.Done.